api/src/routes/rooms/Rooms.ts

```diff
@@ -62,6 +64,18 @@ rooms.post('/', async (req, res) => {
         return;
     }
 
+    // Might be better as a frontend check, but also way more imperformant
+    const goalsNumber = await prisma.goal.count({
```

Comment:
This should be a function in Games.ts in the database folder, something like `goalCount(slug: string)`